mungegithub/github/github.go

```diff
@@ -1216,6 +1218,27 @@ func (obj *MungeObject) AssignPR(owner string) error {
 	return nil
 }
 
+// CloseIssuef will close the given issue with a message
+func (obj *MungeObject) CloseIssuef(format string, args ...interface{}) error {
+	config := obj.config
+	msg := fmt.Sprintf(format, args...)
+	if err := obj.WriteComment(msg); err != nil {
+		return fmt.Errorf("failed to write comment to %v: %q: %v", *obj.Issue.Number, msg, err)
```

Comment:
also, do you know if ClosePR() could be abandoned and use this function instead? I'll try to remember to look sometime.